### PR TITLE
Initial docker log portlayer spec and rest handler

### PR DESCRIPTION
**Pull request by sflxn**
Contributes to #366.

### CODE REVIEW

**Comment by dougm** [comment]
lgtm

File: lib/apiservers/portlayer/restapi/handlers/containers_handlers.go [hunk 1]
```diff
@@ -269,6 +270,14 @@ func (handler *ContainersHandlersImpl) GetContainerListHandler(params containers
 	return containers.NewGetContainerListOK().WithPayload(vmList)
 }
 
+func (handler *ContainersHandlersImpl) GetContainerLogsHandler(params containers.GetContainerLogsParams) middleware.Responder {
+	defer trace.End(trace.Begin("Containers.PostContainerSignal"))
```
Comment (hmahmood) [inline lib/apiservers/portlayer/restapi/handlers/containers_handlers.go]: Is this the right trace message?

**Comment by hmahmood** [comment]
Minor comment ... LGTM

File: lib/apiservers/portlayer/swagger.yml [hunk 2]
```diff
@@ -819,6 +819,51 @@ paths:
...
+          in: query
+          type: boolean
+          default: false
+          required: false
+        - name: since
+          in: query
+          type: integer
+          format: int64
+          required: false
+        - name: timestamp
```
Comment (fdawg4l) [inline lib/apiservers/portlayer/swagger.yml]: This is a docker'ism, right?  For our PL, we don't have to use their (docker's naming).  I'd rather call this "raw".  We could then JSONify the log output and include timestamps in the body unless raw=1.

Just a thought.

**Comment by fdawg4l** [comment]
LGTM

**Comment by caglar10ur** [comment]
LGTM :sheep: :it:

**Comment by sflxn** [comment]
poke

**Pull request merged**